run/grpc-hello-world-mvn/pom.xml

```diff
@@ -80,32 +80,29 @@
       <plugin>
         <groupId>org.xolstice.maven.plugins</groupId>
         <artifactId>protobuf-maven-plugin</artifactId>
-        <version>0.5.0</version>
+        <version>0.6.1</version>
         <configuration>
           <protocArtifact>com.google.protobuf:protoc:${protobuf.version}:exe:${os.detected.classifier}</protocArtifact>
+          <pluginId>grpc-java</pluginId>
+          <pluginArtifact>io.grpc:protoc-gen-grpc-java:${grpc.version}:exe:${os.detected.classifier}</pluginArtifact>
```

Comment:
isn't this invalid as we are no longer using `exe`?

Reply:
That is the gRPC Java protoc plugin which doesn't ship a jar: https://repo.maven.apache.org/maven2/io/grpc/protoc-gen-grpc-java/1.32.1/

Reply:
ahh, missed that. Thanks!